docs/tools/cli/service/kafka-acl.md

```diff
@@ -0,0 +1,71 @@
+---
+title: avn service kafka-acl
+---
+
+Full list of commands for `avn service kafka-acl`.
+
+## Manage Apache Kafka® native access control lists
```

Comment:
```suggestion
## Manage Kafka-native access control lists
```
I have referred to them as Kafka-native ACLs in the documentation. Perhaps we can keep the terminology consistent?